### PR TITLE
Upgrade SMPP Connector Model

### PR DESCRIPTION
**Pull request by PasanT9**
ex. <charset>UTF-8</charset>

> Git Issues: https://github.com/wso2/api-manager/issues/1679, https://github.com/wso2-enterprise/wso2-apim-internal/issues/2116

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <groupId>org.wso2.carbon.connector</groupId>
     <artifactId>org.wso2.carbon.connector.SMPP</artifactId>
     <packaging>jar</packaging>
-    <version>1.0.7</version>
+    <version>1.0.8</version>
```
Comment (SanojPunchihewa) [inline pom.xml:26]: Since we are upgrading the connector model and the SDK, it's better to do a major release like `1.1.0`. @arunans23 WDYT?

Comment (arunans23) [reply pom.xml:26]: Yes. +1

File: src/main/java/org/wso2/carbon/esb/connector/utils/SessionManager.java [hunk 2]
```diff
@@ -59,61 +49,55 @@ private String getKey(String host, int port, String systemId) {
      * @throws IOException
      */
     public SMPPSession getSmppSession(int enquireLinkTimer, int transactionTimer, String host, int port,
-                                      BindParameter bindParameter, int retryCount) throws IOException {
-        SMPPSession smppSession = smppSessionList.get(getKey(host, port, bindParameter.getSystemId()));
+                                      BindParameter bindParameter, int retryCount, String sessionName) throws IOException {
```
Comment (SanojPunchihewa) [inline src/main/java/org/wso2/carbon/esb/connector/utils/SessionManager.java:52]: This seems to exceed maximum line length. Shall we format it?

Comment (PasanT9) [reply src/main/java/org/wso2/carbon/esb/connector/utils/SessionManager.java]: This is fixed with https://github.com/wso2-extensions/esb-connector-smpp/pull/27/commits/3cf3826525c6a63a787dd40ed3a49541d720adfa

File: pom.xml (hunk 1 above)
Comment (PasanT9) [reply pom.xml]: I have updated the connector version to 1.1.0 with https://github.com/wso2-extensions/esb-connector-smpp/pull/27/commits/7e5094daa75e0da673cb023c503d8dd37120aedf

**Review by arunans23 — APPROVED** [review]

**Review by SanojPunchihewa — APPROVED** [review]

**Pull request merged**